gitconfig

```diff
@@ -28,16 +28,23 @@
   template = ~/.gitmessage
 [core]
   excludesfile = ~/.gitignore
+[diff]
+  compactionHeuristic = true
```

Comment:
Hmm, looks like this heuristic is [being deprecated](https://github.com/git/git/commit/3cde4e02ee891bff53bac7f6a7d977f50418a4b5).